Assets/Scripts/CharacterMechanics/CharacterMovement.cs

```diff
@@ -419,8 +438,9 @@ void ApplyMovementVelocity(Vector3 additionalImpulse)
 
         // subtract the component of the move velocity that's going up too steep of a slope
         if (
-            GroundAngle(Controller.height / 2 + dx) > Controller.slopeLimit
+            PointGroundAngle(Controller.height * 1.3f) > Controller.slopeLimit
             && Vector3.Dot(FacingDirection, downSlope) > 0
+            && PointGroundAngle(Controller.height * 1.3f) != 90
```

Comment:
PointGroundAngle probably should not be called twice here (since based on my understanding, Physics.raycast does actually shoot a ray and do that calculation, so probably shouldn't do that twice) (also, 1.3f probably shouldn't be a magic number)